### ISSUE TITLE
No Recipes On Website

### ISSUE THREAD

**Issue by AgentK9** [BCA-Team/Buildcraft-Additions#465]
Yeah, I know, that's what NEI is for, but it is nice for modpack makers who want to tweak recipes and such.

**Comment by AEnterprise** [comment]
problem with that is the time it takes. It takes forever to gather all the screenshots, trim them and put them on the site, if someone can get me the images i can put them on but i don't have much free time and more fun ways to spend it, sorry

**Comment by iTitus** [comment]
I think there is a mod for that purpose.

**Comment by iTitus** [comment]
I don't remember the name though. It generated a pic of every recipe that exists on launch. This is exactly what we need.

**Comment by AgentK9** [comment]
Yeah. I'll do it, but might b a day or 2. (Not the mod, just manually)
